### ISSUE TITLE
Feature Request - Show site usage with more granular time periods

### ISSUE THREAD

**Issue by lakesol** [umbraco/Umbraco.Cloud.Issues#641]
### Issue description

At present umbraco cloud usage shows usage by month to date, can this be enhanced to show a day-by-day analysis?

Thanks

**Comment by sajumb** [comment]
Thanks for the feature request.
We have the data and could provide it as a CSV file download per month. Would that be sufficient?

**Comment by c9mb** [comment]
Apologies for the drive-by comment... personally I think that detailed info should be in the form of downloadable CSV file, and website visualizations limited to summaries.

**Comment by lakesol** [comment]
Yes @sajumb that would work, I would say as long as we get get it for any month shown, not just the current month.

e.g. Today if the 1st April, if an issue was raised today about a site and we wanted to investigate, we may want to see the detail for march.
